Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
```

Comment:
That's fine that you are looking into containerization but according to our task description you shouldn't use docker - we are working with virtual envs. Let's leave docker up to the latest steps.